apitools/base/protorpclite/protojson_test.py

```diff
@@ -198,12 +198,16 @@ def testNumericEnumeration(self):
 
     def testNumericEnumerationNegativeTest(self):
         """Test with an invalid number for the enum value."""
-        self.assertRaisesRegexp(
```

Comment:
Add other tests for the DecodeError that will still be raised?

Reply:
Done. testDecodeInvalidDateTime and testDecodeBadBase64BytesField already did this, added testDecodeInvalidDateTime for a MessageField.